packages/lisk-p2p/src/p2p_types.ts

```diff
@@ -31,10 +31,7 @@ export interface P2PMessagePacket<T> {
 }
 
 export interface P2PNodeStatus {
-	readonly wsPort: number;
-	readonly os: string;
-	readonly version: string;
-	readonly height: number;
+	readonly [key: string]: string;
```

Comment:
why changing this from strongly typed to key-val?

Reply:
While creating an outbound socket we are using it as a query [packages/lisk-p2p/src/peer.ts#L71](https://github.com/LiskHQ/lisk-elements/blob/7bf751af4b4eb90d501db4a80d912e96675fae92/packages/lisk-p2p/src/peer.ts#L71).
And the definition of `query` is coming from  [@types/socketcluster-client](https://www.npmjs.com/package/@types/socketcluster-client),
`query?: string | { [key: string]: string };`

Reply:
Yeah, actually that's why I kept this PR in progress because I wanted to discuss it will @jondubois as well because he wanted to remove `P2PNodeStatus` from here and also from `PeerConfig`. Also, I think we can create a separate type for `query` type and keep the `P2PNodeStatus` as it was before.

Reply:
Maybe, JSON.stringify() will solve the problem? ='(

Reply:
yes it will solve the problem 👍